modules/end-user-guide/partials/proc_creating-a-workspace-from-user-dashboard-quick-add.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// creating-a-workspace-from-code-sample
+
+[id="creating-a-workspace-from-quick-add-view-of-user-dashboard_{context}"]
+= Creating a workspace from Quick Add of user dashboard
+
+This section describes how to create a workspace using the user dashboard and a code sample.
+
+.Prerequisites
+
+* A running instance of {prod-short}. To install an instance of {prod-short}, see xref:installation-guide:installing-che.adoc[].
+
+.Procedure
+
+. Navigate to the {prod-short} dashboard. See xref:navigating-che-using-the-dashboard.adoc[].
+
+. In the left navigation panel, go to *Create Workspace*.
+
+. Click the btn:[Quick Add] tab.
++
+A list of samples to build and run projects are listed in the gallery.
```

Comment:
this sentence doesn't flow very well. How about...

```
The gallery lists sample projects to build and run.
```

or 

```
Sample projects are listed in the gallery.
```

(simpler, since what's the point of a project, if not to build and run it?)

Reply:
Great suggestion. Will do